src/test/ui/proc-macro/capture-macro-rules-invoke.stdout

```diff
@@ -13,8 +14,10 @@ PRINT-BANG INPUT (DEBUG): TokenStream [
 ]
 PRINT-BANG INPUT (DISPLAY): 1 + 1, { "a" }, let a = 1;, String, my_name, 'a, my_val = 30,
 std::option::Option, pub(in some::path) , [a b c], -30
-PRINT-BANG RE-COLLECTED (DISPLAY): 1 + 1, { "a" }, let a = 1, String, my_name, 'a, my_val = 30,
-std :: option :: Option, pub(in some :: path), [a b c], - 30
+PRINT-BANG RE-COLLECTED (DISPLAY): /*⟪*/1 + 1/*⟫*/, /*⟪*/{ "a" }/*⟫*/, /*⟪*/let a = 1/*⟫*/,
```

Comment:
```suggestion
PRINT-BANG RE-COLLECTED (DISPLAY): /*⟪*/ 1 + 1 /*⟫*/, /*⟪*/ { "a" } /*⟫*/, /*⟪*/ let a = 1 /*⟫*/,
```
Looks so noisy.
Maybe the should be given some more space and formatted as braces in `print_mac_common`.

Reply:
Also these are some math symbols, if I copypaste them into notepad++ they look like rectangles (unknown symbols).
```suggestion
PRINT-BANG RE-COLLECTED (DISPLAY): /*«*/ 1 + 1 /*»*/, /*«*/ { "a" } /*»*/, /*«*/ let a = 1 /*»*/,
```
This is how it looks if `LEFT/RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK`s are used instead (which are the oldest unicode bracket symbols).